manifest-gen/src/main.rs

```diff
@@ -25,6 +28,8 @@ use std::{
     path::PathBuf,
 };
 
+const NO_DELETE: [&str; 6] = ["create", "get", "list", "watch", "patch", "update"];
```

Comment:
nit: I have to admit I like to see the list without this constant. It is explicit which operations are allowed